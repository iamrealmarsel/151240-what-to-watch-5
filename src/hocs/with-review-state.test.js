import React from "react";
import TestRenderer from "react-test-renderer";
import withReviewState from "./with-review-state";
import * as mock from "test-mock";


const MockComponent = () => {
  return <section></section>;
};

const MockComponentContainer = withReviewState(MockComponent);

test(`render withReviewState`, () => {
  const tree = TestRenderer
    .create(
        <MockComponentContainer
          postCommentAction={mock.noop}
          match={mock.MATCH}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
