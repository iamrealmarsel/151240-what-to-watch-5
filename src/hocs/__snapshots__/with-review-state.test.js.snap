// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render withReviewState 1`] = `<section />`;
